File: Obnovljivac.API/Models/NasaPower/ParameterInformation.cs
namespace Obnovljivac.API.Models.NasaPower
{
    /// <summary>
    /// Model from 
    /// <see href="https://power.larc.nasa.gov/api/pages/?urls.primaryName=Daily">
    /// POWER Daily API
    /// </see>
    /// <br></br>
    /// The API allows daily data requests of POWER Analysis Ready Data (ARD).
    /// </summary>
    public class ParameterInformation
    {
        public string Units { get; set; }
        public string LongName{ get; set; }
    }
}

File: Obnovljivac.API/ClientApp/src/app/models/autogenerated/parameterInformation.ts

// ----------------------------------------------
//  Interface and model
// ----------------------------------------------
export interface IParameterInformation  {
    units: string;
    longName: string;
}

export class ParameterInformation  implements IParameterInformation {
    public units: string;
    public longName: string;

    constructor(units: string = null,  longName: string = null ) {
        
        this.units = units;
        this.longName = longName;
    }
}

 
